operator/src/operator/pkg/resourcehandler/quobyteclientdaemonset.go

```diff
@@ -7,8 +7,8 @@ import (
 	"time"
 
 	"github.com/golang/glog"
-	"k8s.io/api/apps/v1beta2"
 	v1 "k8s.io/api/core/v1"
+	extentionsv1beta1 "k8s.io/api/extensions/v1beta1"
```

Comment:
You shouldn't use `extenstions/v1beta` this API group is already deprecated see https://kubernetes.io/docs/reference/workloads-18-19

Reply:
That's right, but we require this to be compatible with the client-ds which needs to have k8s 1.6 / openshift 3.6 compabitility at the moment. So its v1beta1 for the moment.

Reply:
Using `extensions/v1beta1` to support earlier kubernetes versions. Currently, all Quobyte deployment daemonsets are using `extensions\v1beta1`.